frontend/src/graphql-components/query/templates.query.graphql

```diff
@@ -0,0 +1,26 @@
+query templates ($workspaceNamespace: String!) {
+   templateList: itPolitoCrownlabsV1alpha2TemplateList(
+    namespace: $workspaceNamespace
+  ) {
+    templates: items 
+    {
+      spec 
+      {
+        name: prettyName
+        description
+        environmentList {
+          guiEnabled
+          persistent
+					resources {
```

Comment:
If would be great it this would be formatted properly, can you check if prettier has some kind of plugin to do it?